jwtauth/jwtauth.go

```diff
@@ -0,0 +1,56 @@
+// Package jwtauth implements an interface for providing CFSSL
+// authentication.
+package jwtauth
+
+import (
+	//"flag"
+	"fmt"
+	//"io"
+	//"io/ioutil"
+	//"os"
+	"time"
+
+	//"github.com/cloudflare/cfssl/helpers"
+	"github.com/dgrijalva/jwt-go"
+)
+
+/*
+ * Not sure if I need these yet
+ *
+type Provider interface {
+	Token(req []byte) (token []byte, err error)
+	Verify(aReq *AuthenticatedRequest) bool
+}
+
+type AuthenticatedRequest struct {
+	// An Authenticator decides whether to use this field.
+	Timestamp     int64  `json:"timestamp,omitempty"`
+	RemoteAddress []byte `json:"remote_address,omitempty"`
+	Token         []byte `json:"token"`
+	Request       []byte `json:"request"`
+}
+*/
+// New attempts to create a new JWT based off of the signing key passed into it
+func New(mySigningKey []byte) (string, error) {
+
+	token := jwt.New(jwt.SigningMethodRS256)
+	token.Claims["exp"] = time.Now().Add(time.Hour * 72).Unix()
+	tokenString, err := token.SignedString(mySigningKey)
+	return tokenString, err
```

Comment:
You could just remove the line above this one and do `return token.SignedString(mySigningKey)`